### PR TITLE
Android dev activity stream

### PR DESCRIPTION
**Pull request by ertbulbul**
Activity stream implemented.

### CODE REVIEW

File: Platon/app/src/main/java/com/cmpe451/platon/page/activity/home/fragment/home/HomeRepository.kt [hunk 1]
```diff
@@ -3,7 +3,10 @@ package com.cmpe451.platon.page.activity.home.fragment.home
 import androidx.lifecycle.MutableLiveData
 import com.cmpe451.platon.network.Resource
 import com.cmpe451.platon.network.RetrofitClient
-import com.cmpe451.platon.network.models.*
```
Comment (darktheorys) [inline Platon/app/src/main/java/com/cmpe451/platon/page/activity/home/fragment/home/HomeRepository.kt:6]: It shows error as "CalendarItem" not imported, can you double check and correct it.

File: Platon/app/src/main/java/com/cmpe451/platon/page/activity/home/fragment/home/HomeViewModel.kt [hunk 2]
```diff
@@ -3,13 +3,16 @@ package com.cmpe451.platon.page.activity.home.fragment.home
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import com.cmpe451.platon.network.Resource
-import com.cmpe451.platon.network.models.*
```
Comment (darktheorys) [inline Platon/app/src/main/java/com/cmpe451/platon/page/activity/home/fragment/home/HomeViewModel.kt:6]: It shows import error "CalendarItem" can you also correct it

**Review by darktheorys — CHANGES_REQUESTED** [review]
If you made the changes i requested, i will approve. Activity stream seems very promisin
g. Good work

**Review by darktheorys — APPROVED** [review]

**Comment by oykuyilmaz1** [comment]
I did not see any errors, thanks

**Review by oykuyilmaz1 — APPROVED** [review]

**Comment by ertbulbul** [comment]
Thanks for all your comments and helps

**Pull request merged**